bot.py

```diff
@@ -16,7 +16,9 @@
 LOGLEVEL = os.environ.get('LOGLEVEL', 'INFO').upper()
 logging.basicConfig(level=LOGLEVEL)
 
-conn = sqlite3.connect('data.db')
+if not os.path.exists('data'):
+    os.makedir('data')
+conn = sqlite3.connect('data/data.db')
```

Comment:
This is a breaking change. I would check for `data.db` existence first, and if it doesn't exist proceed to do the steps (check for the dir || create it) and use the file from that dir.

Or do basically a migration and move that file there.

```python3
if not os.path.exists('data'):
    os.makedir('data')
if os.path.isfile('data.db')
    os.rename('data.db', 'data/data.db')
conn = sqlite3.connect('data/data.db')
```

Reply:
Oh, right. So then it has to check for the existence of data.db first.
```
db_file = 'data.db' if os.path.isfile('data.db') else 'data/data.db'
conn = sqlite3.connect(db_file)
```

Reply:
Only `data/` is exposed to the container, `os.path.isfile('data.db')` will always be false.

Reply:
Not everyone is/will running it in docker tho. 🙂

Reply:
Sure